package/uintent/files/lib/uintent/config.d/30-network.lua

```diff
@@ -59,3 +61,4 @@ for ifname, network in pairs(profile["networks"]) do
 end
 
 uci:commit("network")
+uci:commit("firewall")
```

Comment:
I would prefer dropping the firewall altogether. The AP should be a stupid L2 bridge.

Reply:
I think it is needed for HW offload in the new release

Reply:
Do you have a reference for that?